### ISSUE TITLE
ImporterUI: IndexOutOfBoundsException

### ISSUE THREAD

**Issue by dominikl** [ome/omero-insight#181]
See [29794](https://www.openmicroscopy.org/qa2/qa/feedback/29794/)
```
java.lang.IndexOutOfBoundsException: Index: 0, Tab count: 0
	at javax.swing.JTabbedPane.checkIndex(JTabbedPane.java:1768)
	at javax.swing.JTabbedPane.setSelectedIndex(JTabbedPane.java:589)
	at org.openmicroscopy.shoola.agents.fsimporter.view.ImporterUI.selectChooser(ImporterUI.java:509)
	
```

**Comment by Orohena** [comment]
Hi !
I am the one who raised that issue.
I can now explain how it happen if that help solving the issue:
In Omero insight, I clicked on the "Open the Importer" button.
Because I couldn't find what I was looking for, for some reason, I did a right click on the "tab" that say "Select the data to Import" which opened 3 options "Close", "Close Others", "Close All". I clicked "Close" and Omero crashed.

Hope it will help you understand the issue, anyway, I don't see why someone would try to Close that tab. Even I still don't understand why I did that :)

Flavie
